index.src.html

```diff
@@ -577,24 +581,24 @@ <h3 id="clear-response">
       <a href="https://lists.w3.org/Archives/Public/public-webappsec/2015Jun/0032.html">Martin
       Thomson's public-webappsec post on the topic</a>, for example).
 
-  2.  Let |types| be the result of [[#get-types]] executed on
+  2.  Let |types| be the result of <a>get types</a> executed on
```

Comment:
This is a little too much shorthand: I'd prefer spelling things out in a little more detail: `Let |types| be the result of executing <a>Which data types ought to be removed for |response|?</a>.`